src/content/admin/okteto-insights.mdx

```diff
@@ -173,7 +173,7 @@ Only successful deployments are tracked with `okteto_usage_deploy_duration_secon
   - **`devenv_name`**: Represents the name of the deployed development environment
   - **`namespace`**: Contains the okteto cli context from where the request originated
   - **`repository`**: Contains the git repository where the okteto manifest that defines this development environment is hosted
-  - **`phase`**: Represents the phase of the deployment in the okteto manifest
+  - **`phase`**: Represents the phase of the deployment (either `build` or `commands`) for which the time series is being recorded.
```

Comment:
Don't we have other phases, like `compose` or the phase to wait until all services are healthy?
also, for each command, do we have a different phase?